raiden/tests/fixtures/variables.py

```diff
@@ -51,7 +49,9 @@ def events_poll_timeout():
 @pytest.fixture
 def deposit():
     """ Raiden chain default deposit. """
-    return DEFAULT_DEPOSIT
+    # Arbitrary initial balance for each channel, using a small number to tractable
```

Comment:
Maybe reword the comment and include it in the docstring?
`using a small number to tractable numbers during testing` -> `using a small number for easier calculations during testing`

Or even omit it. It's not really useful information.